src/style.css

```diff
@@ -2,0 +1,49 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src:
+    url(fonts/Avenir-Book.ttf) format('truetype'),
+    url(fonts/Avenir-Heavy.ttf) format('truetype'),
+    url(fonts/Avenir.ttc) format('opentype');
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px #3d4e611a;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: solid 1px #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
```

Comment:
Again, remember to specify a fallback font after 'Avenir' to ensure text remains readable across different platforms and in case the font fails to load.